### ISSUE TITLE
add analog_rca and analog_xlr as distinct input options

### ISSUE THREAD

**Issue by karlfitzke** [amiaopensource/vrecord#404]
Wondering if anyone else using BlackMagic UltraStudio 4K has noticed that as soon as you hit "Return" key to record with vrecord, the XLR audio inputs are selected, regardless of how you set that up in the BlackMagic Desktop Video Setup app that we also use to set input levels.

We can of course use adapter cables, so we always input XLR audio (from RCA deck outputs).  Or to capture stronger input signal levels, I imagine we can work around with RCA to RCA cables by using the BlackMagic Video Setup app to set things back to RCA after recording begins.

Or have I gotten ahead of things here, i.e. are we doing something wrong?

We've seen this behavior with vrecord builds from 2018-08-12 and 2018-12-05.    Using SDI video input.  Matroska 10-bit ffv1 video, analog audio selected.  Thanks.   -Karl

**Comment by dericed** [comment]
vrecord doesn't use the selections of the Desktop Video application. Use `vrecord -e` to get a preference pane where you can select the audio input you prefer.

**Comment by karlfitzke** [comment]
Thanks for the prompt reply, Dave.  

vrecord only allows an "analog" audio input choice, which is why we tried Desktop Video to further clarify which one, RCA or XLR audio, just as we do to successfully set input level (attenuation).  Can selection of either analog RCA or analog XLR audio be added to the To Do list?

**Comment by dericed** [comment]
@karlfitzke so does the BlackMagic UltraStudio 4K has both the analog rca and analog xlr inputs simultaneously on it. The other cards Ive used only have one or the other depending on the breakout cable.

**Comment by karlfitzke** [comment]
Yes, the 4K has both RCA and XLR analog audio inputs - but only two channels each, and one or the other pair (RCA or XLR) selectable at any given time with the software that I have, and likely hardware too to save money and back panel space - read on if interested in hearing more. 

Only having two audio input channels of course assumes, sadly to us archival people, that no one is capturing old four channel video tapes anymore.   We use the 4K for two channel video only as a result.

The four possible inputs on our Studio 2 and breakout cable are physically 1/4" TRS (tip-ring-sleeve), but nonetheless "balanced" audio inputs that are electrically the same as XLR.   We have adapter cables for the Studio 2 setup to do what we need to, e.g. go from RCA ("single-ended" electrically instead of balanced) to TRS, or XLR to TRS.  We can do the same for RCA to XLR on the 4K, but input levels are then weaker, thanks to different specs on nominal levels for balanced vs single-ended audio.  Perhaps you've seen choices of +4/-10 audio levels on gear or in software, which is what that is usually about.

**Comment by dericed** [comment]
i changed the title, citing http://ffmpeg.org/ffmpeg-devices.html#decklink for reference

**Comment by karlfitzke** [comment]
Thanks.  Looks promising that the ffmpeg options are already there.

**Comment by privatezero** [comment]
Going to close this as it has been a while. Reopen if this is still desired as a potential addition!
